frontend/src/layouts/MainLayout.js

```diff
@@ -85,13 +75,7 @@ const PersistentDrawerLeft = props => {
           paper: classes.drawerPaper,
         }}
       >
-        <div className={classes.drawerHeader}>
-          <IconButton onClick={handleDrawerClose}>
-            <ChevronLeftIcon />
-          </IconButton>
-        </div>
```

Comment:
I moved the drawerHeader div into the Sidebar component so that the select menu and the close arrow could be positioned next to each other.